conda_forge_tick/git_utils.xsh

```diff
@@ -136,12 +136,24 @@ def get_repo(attrs, branch, feedstock=None, protocol='ssh',
     return feedstock_dir, repo
 
 
+def delete_branch(pr_json: LazyJson):
+    token = $PASSWORD
+    deploy_repo = $USERNAME + '/' + pr_json['base']['name']
+    doctr_run(['git', 'push', 'https://{token}@github.com/{deploy_repo}.git'.format(
```

Comment:
probably need to have a checkout of this repo around,  that we can dispose of afterwards